docs/modules/ROOT/examples/calculator-client/pom.xml

```diff
@@ -4,7 +4,7 @@
     <parent>
         <groupId>io.quarkiverse.cxf</groupId>
         <artifactId>quarkus-cxf-integration-tests</artifactId>
-        <version>3.9.1-SNAPSHOT</version>
+        <version>3.10.0-SNAPSHOT</version>
```

Comment:
@ppalaga I had to update this version, to make CI pass. I think that either:

- control of version is too strict for this example
- release script has to be changed to include this file for version upgrade

Reply:
Yeah, it's pretty stupid. I have pushed a "Fixup release" commit manually in the past. Now I have implemented a proper fix in  https://github.com/quarkiverse/quarkus-cxf/commit/23ff3436f12f67f1459a87b9d82e111a65c6c05b